buildbreaker.properties

```diff
@@ -0,0 +1,13 @@
+category=Integration
+description=Break the analyzer if the project does not pass its Quality Gate
```

Comment:
Technically, this "fails the analysis" rather than _breaking_ the _analyzer_. An update here would be nice.

Reply:
Fair point, the wording is a bit strange. I updated it to your suggestion.